index.html

```diff
@@ -18,6 +18,8 @@ <h1>CSS lessons by Sorax</h1>
         <a href="06/index.html"><li class="link">Lesson 04</li></a>
         <a href="07/index.html"><li class="link">Lesson 05</li></a>
         <a href="08/index.html"><li class="link">Lesson 06</li></a>
+        <a href="09/index.html"><li class="link">Lesson 07</li></a>
+        <a href="10/index.html"><li class="link">Lesson 08</li></a>
```

Comment:
Class "link it is not correct name for this selector because actually you give this class not for link, but for list item